serveradmin/servershell/static/js/servershell/command.js

```diff
@@ -235,6 +235,31 @@ servershell.commands = {
         // Now trigger reload with all changes at once
         servershell.shown_attributes = shown_attributes;
     },
+    bookmarkadd: function(name) {
+        if (!name) {
+            servershell.alert('Missing name for bookmark', 'warning');
+            return;
+        }
+
+        localStorage.setItem(name, servershell.shown_attributes);
```

Comment:
I guess it would already make sense to avoid conflicts later in the localStorage to prefix them all with a constant:
``` js
let key = "bookmark-" + name`
```
When the localStorage will be used for something else, you don't end up in mixing the values.
Then you would need to check for the prefix though by traversing through the items.
It seems that some are also using it like that:
``` js
let bookmarks = localStorage.getItem("bookmarks")
bookmarks.add(newBookmark)
localStorage.setItem("bookmarks", JSON.stringify(bookmarks))
```
So, you use more complex data inside one key. With that you don't need to traverse anymore through the items of the localStorage but just directly fetch the item data you need (and need to traverse that then).

One of these I would already do, so it is future safe.

Reply:
Hmm yeah that probably makes sense.

Reply:
Added `bookmark.` prefix everywhere.